### PR TITLE
feat: add pool modules to symbiosis finance

### PR DESCRIPTION
**Pull request by allush**
- **Project Name**:

- **Project Website**:

- **Description**: (New Project, new Feature for a project, bugfixes, etc...)

- **Please check if the PR fulfills these requirements**

* [ ] The projects name, twitter, URL etc have been filled out in `index.ts`
* [ ] Tests for the new features have been added (for bug fixes / features)
* [ ] Any required Logos are included in this PR

- **Logo**: (High res logo compatible with dark and light backgrounds)

- **Other information**: (Any farms or features left to be implemented, addresses to be used for testing, etc)

### CODE REVIEW

**Review by witem — COMMENTED** [review]
So I general code looks good. But I have questions:
1. Are these LP pools or farming positions? Because naming for apr using `farming` URL, but `type: 'pools'` we're using only for LP positions
2. What does the user receive when creating a position for this feature? Does he receive some LP tokens? And on what chain, if yes?

File: apps/workshop/projects/SymbiosisFinance/modules/pools.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      if (!token) return [];
+
+      const balance = await getUserPosition(context, currentPool, user);
+
+      return [
+        {
+          id: pool.id,
+          supplied: [
+            {
+              token,
```
Comment (witem) [inline apps/workshop/projects/SymbiosisFinance/modules/pools.ts:74]:
```suggestion
              ...pool.supplied[0],
```

Comment (allush) [reply apps/workshop/projects/SymbiosisFinance/modules/pools.ts]: fixed, thanks!

i have made same change in vesis module

**Comment by allush** [comment]
> So I general code looks good. But I have questions:
> 
> 1. Are these LP pools or farming positions? Because naming for apr using `farming` URL, but `type: 'pools'` we're using only for LP positions
> 2. What does the user receive when creating a position for this feature? Does he receive some LP tokens? And on what chain, if yes?

1. These are LP pools positions. I have changed url. We provide cross chain farmin to all LP providers in our multistable pool.

2. When user deposits USDC on Ethereum USDC lock on Ethereum chain in Portal contract, then his tokens bridge to BOBA BNB chain and deposit to multistable pool. User receives LP tokens on BOBA BNB chain.

**Review by witem — APPROVED** [review]

**Comment by allush** [comment]
@witem Strange problem with formatting. All pre-commit checks have passed successfully.
Workshop's prettier style differs from main prettier style again.

**Comment by witem** [comment]
@allush there is one prettier config in project

**Comment by allush** [comment]
> @allush there is one prettier config in project

When I run `yarn workshop format --write` it changes code so:
Screenshot from 2023-02-09 12-51-34

When I try to git commit from main folder of repo it shows there are no changes. How can it be?)
Screenshot from 2023-02-09 12-52-10

How to commit correctly?

**Comment by witem** [comment]
@allush found a problem in https://github.com/defiyield-app/defiyield-self-integrations/pull/29 . Can you please try again?

**Comment by allush** [comment]
> @allush found a problem in #29 . Can you please try again?

same problem.

commited as `git commit --no-verify`...

**Comment by witem** [comment]
@allush, can you please share a few wallet addresses with active positions in different pools?

**Comment by allush** [comment]
> @allush, can you please share a few wallet addresses with active positions in different pools?


`0x3254aE00947e44B7fD03F50b93B9acFEd59F9620`
`0x04fDE790D26Da69AA9178249EAD6863977684e9D`

**Comment by witem** [comment]
Thanks @allush 
Is this looks as expected https://development.defiyield.app/account/0x04fde790d26da69aa9178249ead6863977684e9d/dashboard/portfolio-overview ?
image

**Comment by allush** [comment]
> Thanks @allush Is this looks as expected https://development.defiyield.app/account/0x04fde790d26da69aa9178249ead6863977684e9d/dashboard/portfolio-overview ? image

Yes, perfect!

**Pull request merged**